src/v2/view-builder/views/ov/EnrollChannelPollDescriptionView.js

```diff
@@ -257,9 +257,9 @@ function getDeviceMap(appState) {
   const contextualData = appState.get('currentAuthenticator').contextualData;
   let deviceMap = {};
   if (contextualData.samedevice?.setupOVUrl) {
-    deviceMap = contextualData.samedevice;
+    deviceMap = {...contextualData.samedevice};
   } else if (contextualData.devicebootstrap?.setupOVUrl) {
-    deviceMap = contextualData.devicebootstrap;
+    deviceMap = {...contextualData.devicebootstrap};
```

Comment:
make a copy of `contextualData.samedevice/bootstrap` instead of editing the original transaction in appState, since this was causing the prevTransaction and new transaction to not be identical, causing the SIW the re-render on every poll.